llvm/lib/Support/APFloat.cpp

```diff
@@ -139,6 +143,10 @@ static constexpr fltSemantics semFloat8E4M3FNUZ = {
 static constexpr fltSemantics semFloat8E4M3B11FNUZ = {
     4, -10, 4, 8, fltNonfiniteBehavior::NanOnly, fltNanEncoding::NegativeZero};
 static constexpr fltSemantics semFloatTF32 = {127, -126, 11, 19};
+static constexpr fltSemantics semFloat6E3M2FN = {
```

Comment:
Naming bikeshed: this seems to overload `FN` from E4M3, where it means Finite, but has NaN.

But maybe that's not that big a deal

Reply:
Yes, I tried a few other suffixes like "FNOnly", "FNNoNaN" etc. They looked weird and lengthy. So, I stayed with the "FN" suffix.